python/bridgestan/model.py

```diff
@@ -81,9 +74,7 @@ def __init__(
         self._free_error.argtypes = [ctypes.c_char_p]
 
         err = ctypes.pointer(ctypes.c_char_p())
-        self.model_rng = self._construct(
-            str.encode(self.data_path), self.seed, self.chain_id, err
-        )
+        self.model_rng = self._construct(str.encode(self.data_path), self.seed, err)
```

Comment:
Should we change `self.model_rng` to `self.model` to better reflect the changes at the C API level?

Reply:
Done, good catch